examples/dash/dash-connect-pgsql/README.md

```diff
@@ -0,0 +1,38 @@
+# Dash App Connected to PostgreSQL Database
+
+Interactive Dash Application, connected to PostgreSQL database.
+
+![](app.png)
+
+## Set up local testing environment
+To use the app, store the below information locally into your `.env` for `upload.py`. These variables can be found on the `Parameters only` section under `connection details` from your [Neon](https://console.neon.tech/) PostgreSQL dashboard.
+```
+PGHOST='YOUR_HOST'
+PGDATABASE='test'
+PGUSER='test_owner'
+PGPASSWORD='your_password'
+```
+
+Run `python -m pip install -r requirements.txt` to install all necessary packages.
+
+## Upload dataset to your Postgres server
```

Comment:
I just realised that the dataset link is not mentioned here. From the code it looks like the csv files should be saved in a `data` folder. please mention that as well @bchen39

Reply:
[added](https://github.com/ploomber/doc/pull/259/commits/e0f5d8c8102adada3cb512c13c0117d12cb18541#diff-947610ca10aacde7e2a54afa9c3fbd37fea9055230c4a6222dc39decedc110afR19)

Reply:
@bchen39 please add a link to the datasets, otherwise users don't know where to download the data from.

Reply:
I wrote a program to extract a portion of the data from the datasets. Should I include that program as well?

Reply:
[link to the datasets](https://github.com/ploomber/doc/pull/259/commits/06bd1957e7ddf77928a24bc9438a47110379fd3e#diff-947610ca10aacde7e2a54afa9c3fbd37fea9055230c4a6222dc39decedc110afR19). I also added instruction on how to obtain my extracted dataset, together with the corresponding [program](https://github.com/ploomber/doc/pull/259/commits/06bd1957e7ddf77928a24bc9438a47110379fd3e#diff-8021001209dc386f683f2e962a504491e43c1b83bb717cd644d5e98d4eb5d79e).